spec/controllers/courses_controller_spec.rb

```diff
@@ -165,6 +165,88 @@
       get_as instructor, :show, params: { id: course }
       expect(response.status).to eq(200)
     end
+    it 'responds with success on edit' do
+      get_as instructor, :edit, params: { id: course }
+      expect(response).to have_http_status(200)
+    end
+
+    context 'updating course visibility' do
+      context 'as an authorized instructor' do
+        it 'responds with success on update' do
+          put_as instructor, :update,
+                 params: { id: course.id, course: { name: 'CS101', display_name: 'Intro to CS', is_hidden: false } }
+          expect(response).to have_http_status(302)
+        end
+        it 'successfully updates when parameters are valid' do
```

Comment:
This description should be more specific: only the `is_hidden` attribute is updated, but not the others.